source/examples/rapids-autoscaling-multi-tenant-kubernetes/notebook.ipynb

```diff
@@ -3,7 +3,17 @@
   {
    "attachments": {},
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "tags": [
+     "platform/kubernetes",
+     "cloud/gcp/gke",
+     "tools/dask-operator",
+     "dataset/nyc-taxi",
```

Comment:
What other dataset tags do we currently have? Curious because not sure how I feel about tagging datasets too

Reply:
I'm going to merge these changes and open a new issue to track this conversation.